pkg/cmd/openshift-apiserver/openshiftapiserver/openshift_apiserver.go

```diff
@@ -509,6 +511,29 @@ func (c *completedConfig) withUserAPIServer(delegateAPIServer genericapiserver.D
 	return server.GenericAPIServer, &legacyStorageVersionMutator{version: userapiv1.SchemeGroupVersion, storage: storage}, nil
 }
 
+func (c *completedConfig) withOpenAPIAggregator(delegatedAPIServer *genericapiserver.GenericAPIServer) (*genericapiserver.GenericAPIServer, error) {
+	c.GenericConfig.RemoveOpenAPIData(delegatedAPIServer)
+
+	specDownloader := openapicontroller.NewDownloader()
+	openAPIAggregator, err := openapicontroller.BuildAndRegisterAggregator(
+		&specDownloader,
+		delegatedAPIServer,
+		delegatedAPIServer.Handler.GoRestfulContainer.RegisteredWebServices(),
+		configprocessing.DefaultOpenAPIConfig(nil),
```

Comment:
comment that it's fine to start with an empty spec because we don't care about oapi entries in it

Reply:
Probably we have to apply all OpenShift customizations to this spec as we had before in the monolithic start command.

Reply:
> comment that it's fine to start with an empty spec because we don't care about oapi entries in it

Thanks, done